pages/desktop/user.py

```diff
@@ -89,7 +90,7 @@ class EditProfile(Base):
     _profile_locator = (By.CSS_SELECTOR, "#profile-personal > legend")
     _details_locator = (By.CSS_SELECTOR, "#profile-detail > legend")
     _notification_locator = (By.CSS_SELECTOR, "#acct-notify > legend")
-    _hide_email_checkbox = (By.ID, 'id_emailhidden')
+    _email_checkbox = (By.ID, 'id_emailhidden')
```

Comment:
The variable should have the `_locator` suffix as do all the others. Also, I actually prefer the old name. This checkbox is checked when the user wants to hide their email address, correct? If that's the case then just `email_checkbox` is less accurate than `hide_email_checkbox`.